.omeroci/scripts-copy

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+set -u
+set -x
+
+export TARGET=${TARGET:-..}
+export OMERO_DIST=${OMERO_DIST:-/opt/omero/server/OMERO.server}
```

Comment:
We could likely add these environment variables to the "API" of these files, probably to be specified in a README or `docs/`